src/riak_kv_pb_object.erl

```diff
@@ -190,8 +193,8 @@ process(#rpbputreq{bucket=B0, type=T, key=K, vclock=PbVC,
     case Result of
         consistent ->
             process(Req#rpbputreq{if_not_modified=undefined,
-                                  if_none_match=consistent},
-                    State);
+                                  if_none_match=undefined},
```

Comment:
Haha, well it's done now anyway 😄 . Any other review comments, or are you ready to give it a +1?